site/docs/v1/tech/identity-providers/samlv2/azure-ad.adoc

```diff
@@ -0,0 +1,179 @@
+
+image:identity-providers/azure-ad-saml/new-login-button[New Login With Azure-Button,width=1200,role=shadowed]
+
+== Create an Application in Azure
+
+[NOTE]
+====
+For this guide, we use Azure Single Sign-on functionality. There may be alternate methods to connect SAML v2 and AzureAD, but these would fall outside of the scope of this documentation.
+====
+
+If you are following this guide, likely you have already configured an AzureAD Enterprise application. If you have not, please follow the brief steps outlined below:
+
+1. From the link:https://portal.azure.com/#home[account portal] navigate to `Enterprise Applications`.
+2. At the top of the screen click on `New application`
+3. Click on `Create your own application`
+4. Name the application
+5. Select the third option - `Integrate any other application you don't find in the gallery (Non-gallery)`.
+6. Click `Create`
+
+image:identity-providers/azure-ad-saml/create-non-gallery-application[Create Azure AD Application,width=1200,role=shadowed]
+
+== Configure Your Azure Application
+
+From your application home screen, click on Single sign-on. For this integration please select the SAML option.
+
+image:identity-providers/azure-ad-saml/configure-single-sign-on-saml[Configure Single-sign on saml,width=1200,role=shadowed]
+
+AzureAD will present five steps to complete. However, we have a bit of a procedural dilemma. Before we can move forward, we need to make an Entity ID (our FusionAuth Idp, in this case) so AzureAD can generate a X509 certificate. However, before we make an Entity ID/IdP in FusionAuth, we need an X509 certificate. The below steps will solve this problem and complete our integration. In sum, we will be:
+
+1. Creating a "dummy" X509 certificate and importing this into Key Master.
+2. Making a SAML Idp in FusionAuth (Entity Id) with a dummy certificate selected.
+3. Updating our AzureAD Enterprise application with values from our newly created Idp, thereby getting a valid X509 certificate.
+4. Revisiting our SAML Idp in FusionAuth and updating with the correct X509 certificate from the above step.
+5. Completing our integration.
+
+If you already have a valid X509 certificate, you can complete step two above using this valid certificate and skip a few of the steps below.
+
+With that discussed, let us begin!
+
+== Import a "Dummy" X509 Certificate
+
+The first step is to import a "dummy" certificate into Key Master. You can generate any X509 certificate as it will be changed later. Once generated, under Key Master (`Settings > Key Master`) click `Import Certificate`.
+
+image:identity-providers/azure-ad-saml/import-certificate[Import Certificate,width=1200,role=shadowed]
+
+Included is an example X509 certificate for sample use.
```

Comment:
Ah, so they don't even need to create one, they can just use this one? Any reason why we wouldn't just tell folks to import this one (and not even mess with generation)?

Reply:
Good point!

Reply:
Was considering this, but thought that it would be a bit presumptious that they would use my certificate 😄